src/utils/mathUtils.ts

```diff
@@ -75,6 +75,10 @@ export module Utils {
       return typeof n === "number" && !Plottable.Utils.Math.isNaN(n) && isFinite(n);
     }
 
+    /**
+     * Generates an array of consecutive, strictly increasing numbers
+     * in the range [start, stop) at a step distance of each other
```

Comment:
Yes, although if you didn't get it means I was too subtle. [ ] generally denote closed interval, ( ) generally denote open interval